pkg/utils/status.go

```diff
@@ -419,3 +431,78 @@ func WaitForNginxController(ctx context.Context, instance csmv1.ContainerStorage
 
 	return wait.PollImmediate(time.Second, timeout, GetNginxControllerStatus(ctx, instance, r))
 }
+
+// checkForServices - Calculate success state for services deployed with app-mob
+func checkForServices(ctx context.Context, instance *csmv1.ContainerStorageModule, r ReconcileCSM, newStatus *csmv1.ContainerStorageModuleStatus, isDaemonset bool) (bool, error) {
+
+	log := logger.GetLogger(ctx)
+	running := false
+	var err error = nil
+	if isDaemonset {
+		expected, nodeStatus, daemonSetErr := getDaemonSetStatus(ctx, instance, r)
+		newStatus.NodeStatus = nodeStatus
+		newStatus.State = constants.Failed
+		log.Infof("daemonset expected [%d]", expected)
+		log.Infof("daemonset nodeStatus.Available [%s]", nodeStatus.Available)
+		if fmt.Sprintf("%d", expected) == nodeStatus.Available {
+			running = true
+			newStatus.State = constants.Succeeded
+		}
+		if daemonSetErr != nil {
+			err = daemonSetErr
+			log.Infof("calculate Daemonseterror msg [%s]", daemonSetErr.Error())
+		}
+
+	} else {
+		controllerReplicas := newStatus.ControllerStatus.Desired
+		controllerStatus := newStatus.ControllerStatus
+		newStatus.State = constants.Failed
+		log.Infof("deployment controllerReplicas [%s]", controllerReplicas)
+		log.Infof("deployment controllerStatus.Available [%s]", controllerStatus.Available)
+		if controllerReplicas == controllerStatus.Available {
+			running = true
+			newStatus.State = constants.Succeeded
+		}
+	}
+	log.Infof("calculate overall state [%s] of module", newStatus.State)
+
+	return running, err
+
+}
+
+// statusForAppMob - calculate success state for application-mobility module
+func statusForAppMob(ctx context.Context, instance *csmv1.ContainerStorageModule, r ReconcileCSM, newStatus *csmv1.ContainerStorageModuleStatus) (bool, error) {
+
+	//log := logger.GetLogger(ctx)
+	running := false
+	var AppRunning bool
```

Comment:
I believe variables in golang are only supposed to be capitalized if they are meant to be exported -- can we fix that?

Reply:
i thought you needed camelcase for varibales
you provided the feedback in the last PR

Reply:
i have changed it

Reply:
i have changed it to be appRunning